packages/blade/src/components/ProgressBar/_decisions/decisions.md

```diff
@@ -30,7 +30,7 @@ A Progress bar is generally a branded element that indicates progress of process
 | size | `small`, `medium` | `small` | The size of the progress bar to be rendered. |  |
 | accessibilityLabel | `string` | `undefined` | The accessibility label (aria-label) for the progress bar. |
 | isIndeterminate | `boolean` | `false` | Control whether the progress bar is indeterminate or not. The progress bar will remain in an indeterminate state and respect the `isIndeterminate` prop if `isIndeterminate` is passed along with `value`. This prop will be ignored if `variant` is `meter`  |
-| showPercentage | `boolean` | `true` | Control whether the progress bar should show the current progress percentage or not. Percentage would be automatically hidden when the progress bar is in an indeterminate state.  |
+| hidePercentage | `boolean` | `false` | Control whether the progress bar should show the current progress percentage or not. Percentage would be automatically hidden when the progress bar is in an indeterminate state.  |
```

Comment:
Making it true by default is a bit tricky since undefined is also a valid value. If we use `showPercentage`, the consumers will have to explicitly pass `true` or `false` whereas if we use `hidePercentage`, the absence of that prop indicates that its unset hence false.

Reply:
For example we have `showClearButton` and consumers pass it only when they want to change the default behaviour and show it since clear button is hidden by default. Whereas in terms of `hidePercentage` the default behaviour is that it's visible and a prop needs to be passed to hide it. A consumer will think of this prop when the want to _hide_ the percentage only. So asking them to pass a negation `showPercentage={false}` seems counterintuitive.

Reply:
I find `showSomething` easier to parse. We could keep the default value as `true`. Some examples we've right now in other components - `showRevealButton`, `showClearButton` which are default `true`.